src/components/Swap/Toolbar/index.tsx

```diff
@@ -113,8 +139,21 @@ export default memo(function Toolbar() {
   }
 
   return (
-    <ToolbarRow flex justify="space-between" data-testid="toolbar">
-      {caption}
-    </ToolbarRow>
+    <ToolbarColumn>
+      <ToolbarRow flex justify="space-between" data-testid="toolbar">
+        {caption}
+      </ToolbarRow>
+      {expanded && (
```

Comment:
Shouldn't expanded only be available for a valid trade?

Reply:
this is out of date now - but i don't think we should assume that we need a valid trade to show the summary / routing info components - they should be able to handle an invalid trade and show a pending state